core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UserCoreConstants.java

```diff
@@ -129,6 +129,8 @@
         public static final String LOCAL_NAME_EVERYONE_ROLE = "EveryOneRoleName";
         public static final String LOCAL_NAME_ANONYMOUS_USER = "AnonymousUser";
         public static final String LOCAL_PASSWORDS_EXTERNALLY_MANAGED = "PasswordsExternallyManaged";
+        public static final String OVERRIDE_USERNAME_CLAIM_FROM_INTERNAL_USERNAME =
+                "overrideUsernameClaimFromInternalUsernameElem";
```

Comment:
the element name is 'OverrideUsernameClaimFromInternalUsername'

Reply:
done